### PR TITLE
Change to the Trusty distro

### PR DESCRIPTION
**Pull request by khellang**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +10,6 @@ install:
   - echo set -e >> ./travis.sh
   - echo set -x >> ./travis.sh
   - echo set -o pipefail >> ./travis.sh
-  - echo echo Running on Mono 3.x >> ./travis.sh
```
Comment (Yantrio) [inline .travis.yml]: maybe just change this to `echo mono --version >> ./travis.sh` to get the mono version up to date.

**Comment by Yantrio** [comment]
I'm waiting

File: .travis.yml (hunk 1 above)
Comment (khellang) [inline .travis.yml]: Not necessary. Travis already spits out the mono version when building :smile:

Comment (Yantrio) [inline .travis.yml]: ok, :shipit:

**Pull request merged**